public/plugin_nav.tsx

```diff
@@ -33,47 +32,25 @@ export function registerAllPluginNavGroups(core: CoreSetup<AppPluginStartDepende
   core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
     {
       id: observabilityNotebookID,
-      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
       order: 400,
     },
   ]);
   core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS[`security-analytics`], [
     {
       id: observabilityNotebookID,
-      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
       order: 400,
     },
   ]);
   core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.analytics, [
     {
       id: observabilityNotebookID,
-      category: DEFAULT_APP_CATEGORIES.dashboardAndReport,
+      category: DEFAULT_APP_CATEGORIES.visualizeAndReport,
       order: 400,
     },
   ]);
 
-  core.chrome.navGroup.addNavLinksToGroup(DEFAULT_NAV_GROUPS.observability, [
```

Comment:
Where is integrations moving to?

Reply:
Just saying in observability now removed from security and analytics